core-bundle/src/Resources/config/voters.yml

```diff
@@ -0,0 +1,18 @@
+services:
+    _defaults:
+        autoconfigure: true
+
+    contao.security.backend_access_voter:
```

Comment:
I don‘t like these services to be in a separate file. If at all, they should be in a `security.yml` file because that is the name of the namespace.

Would you mind me moving the definitions back to the `services.yml` file?

Reply:
The number of voters will grow considerably in the future. That's why. I think it will become quite a mess in `services.yml`.

Reply:
Then we should think of using a dedicated namespace (e.g. `contao.security.voter.`) for the voters, so we can move them to a separate `voters.yml` file in the future.

Reply:
Yeah but then the service name linter will fail. But anyway, you can totally move them to wherever you prefer. I'm just saying that the number will grow significantly and it's probably better to have those in their own file for easier lookup :)

Reply:
As discussed in Slack, we want to keep the service definitions in the `services.yml` for now and discuss this again when the other voters are added.